core/commands/files/files.go

```diff
@@ -449,6 +449,7 @@ Example:
 		cmds.BoolOption("e", "create", "create the file if it does not exist"),
 		cmds.BoolOption("t", "truncate", "truncate the file before writing"),
 		cmds.IntOption("n", "count", "maximum number of bytes to read"),
+		cmds.BoolOption("f", "no-flush", "sync only this file after write"),
```

Comment:
can we use `--flush` which just defaults to true? these `--no-X` flags will be confusing, it would  be nice to opt for always `--X` with defaults to `--X=true`